src/make_data/fake-data.R

```diff
@@ -334,12 +332,16 @@ nhs_list <- sample_frac(master, nhs_perc + nhs_and_web_perc)
 web_list <- sample_frac(master, web_perc + nhs_and_web_perc)
 ivr_list <- sample_frac(nhs_list, ivr_perc)
 
-nhs_list <- select_at(nhs_list, nhs_column_names)
-web_list <- select_at(web_list, web_column_names)
+# nhs_list <- select_at(nhs_list, nhs_column_names)
+# web_list <- select_at(web_list, web_column_na
```

Comment:
Are we deliberately commenting these out? If so, we aren't able to apply the new column name, `Flag_HaemotologicalCancers` to the current name, `Flag_HaemotologicalCancers`, as per the request.

```suggestion
nhs_list <- select_at(nhs_list, nhs_column_names)
web_list <- select_at(web_list, web_column_na
```

Reply:
:man_facepalming: